src/hooks/useHailFrequency.js

```diff
@@ -66,16 +66,27 @@ async function fetcher(query, spaceId, spaceToken) {
  * @param {function(Object[]):Object[]} params.dataHandler - params.dataHandler - Data handler function that can be used to transform data returned from Nacelle's indices
  * @returns {Object[]} The data stored in Nacelle's indices
  */
-export const useHailFrequency = ({ query, options, dataHandler = (data) => data }) => {
-
+export const useHailFrequency = ({
+  query,
+  options,
+  type,
+  dataHandler = (data) => data
+}) => {
   let spaceId =
     config.nacelleSpaceId || process.env.SANITY_STUDIO_NACELLE_SPACE_ID
   let spaceToken =
     config.nacelleSpaceToken || process.env.SANITY_STUDIO_NACELLE_SPACE_TOKEN
-  if(options && options.spaceId) spaceId = options.spaceId
-  if(options && options.spaceToken) spaceToken = options.spaceToken
+  const endpoint =
```

Comment:
I used the nullish coalescing operator & optional chaining here because I think it's a bit easier to reason about than the inline if's. I should probably go do that with the other options so they're all consistent 🤔